gear-up.md

```diff
@@ -0,0 +1,26 @@
+* What role does empathy play in your life and how has it helped you?
```

Comment:
Hey Johnathan! We'd like to see you expand on these responses. We expect about a paragraph in response to each question. Please update these to be more robust. Your last response has the level of effort we are looking for.